src/layer/DivOverlay.js

```diff
@@ -19,16 +19,16 @@ export var DivOverlay = Layer.extend({
 	// @aka DivOverlay options
 	options: {
 		// @option offset: Point = Point(0, 7)
-		// The offset of the popup position. Useful to control the anchor
+		// The offset of the overlay position. Useful to control the anchor
 		// of the popup when opening it on some overlays.

```

Comment:
> when opening it on some overlays

What does it ever mean?

Reply:
I don't know. Maybe we can remove the second sentence